test/e2e/certificates_test.go

```diff
@@ -354,6 +355,50 @@ var _ = Describe("ACME Certificate", Ordered, func() {
 			Expect(err).NotTo(HaveOccurred())
 		})
 	})
+
+	Context("OCP-65035-CLOUD_CREDENTIALS_SECRET_NAME works well in mint/passthrough clusters", func() {
+		It("should obtain a valid LetsEncrypt certificate", func() {
+
+			By("getting Infrastructure object")
+			infra, err := configClient.Infrastructures().Get(ctx, "cluster", metav1.GetOptions{})
+			Expect(err).NotTo(HaveOccurred())
+			if infra.Status.PlatformStatus.Type != configv1.GCPPlatformType {
+				Skip("Skip this case for current plaform, it's not gcp.")
+			}
+
+			By("Check cloud credential in cluster")
+			_, err = loader.KubeClient.CoreV1().Secrets("cert-manager").Get(ctx, "gcp-credentials", metav1.GetOptions{})
+			Expect(err).NotTo(HaveOccurred())
```

Comment:
@geliu2016 , like we do for AWS https://github.com/openshift/openshift-tests-private/blob/7b0a42543a777413c09289ea1d95b32fcacb06b5/test/extended/apiserverauth/certmanager.go#L56 , here we need to skip, instead of assert `Expect(err).NotTo(HaveOccurred())`.

Reply:
Updated.